Sources/Auth/Services/Signer/MessageSigner.swift

```diff
@@ -0,0 +1,36 @@
+import Foundation
+
+protocol MessageSignatureVerifying {
+    func verify(signature: String, message: String, address: String) throws
+}
+
+protocol MessageSignatureSigning {
+    func sign(message: String, privateKey: Data) throws -> String
+}
+
+struct MessageSigner: MessageSignatureVerifying & MessageSignatureSigning {
```

Comment:
Would make sense to split it into two separate classes?
SIWESigner and SIWEVerifier?
and maybe throw an error if message argument in function call is not a valid SIWE message?

Reply:
This is not a real SIWESigner/Verifier. It will for property for regular strings too. I think need to live it in a single class. Splitted protocols just encapsulate methods usage in app/wallet parts